.github/workflows/ci.yml

```diff
@@ -55,7 +55,7 @@ jobs:
 
       - run: npm ci
       - run: npm run lint
-      - run: npm run package
+      - run: npm run package -- --all
```

Comment:
Instead of testing all electron options on each run, would it make sense to restrict this to the host platform?
(I.e. only build win32 electron on Windows...)

Reply:
We could even just use `npm run build` and only for the node `10` build which currently publishes the electron artifacts run `npm run package` to speed things up.
When I first integrated it I wanted to make sure that this runs on all OSs for all packages but maybe we can remove that now.
This also sometimes hangs the macOS build.

Reply:
I would be in favor of speeding things up.